util/cron/nightlysubs.pm

```diff
@@ -117,7 +117,11 @@ sub endMailHeader {
 }
 
 sub endMailChplenv {
-    my $chplenv = `$chplhomedir/util/printchplenv --debug`;
+    my $ch = $chplhomedir;
```

Comment:
Actually, it looks like util/test/perf/ does exist in the tarball, but if we're testing against the chapel 1.6 tarball it may not be the one we want to use.